### ISSUE TITLE
Change SSL Mode for postgres

### ISSUE THREAD

**Issue by jandr-dfds** [dfds/infrastructure-modules#376]
Hi

**TL;DR;** Add support for changing `SSL Mode` for the connection string that is stored in parameter store. Currently it is hardcoded to `SSL Mode=Require`, but we need to be able to change it.

When developing .NET applications using Postgres we tend to use the [Npgsql](https://www.npgsql.org/) library. A breaking change around `SSL Mode` has been introduced to **v6** of that library (read more here: [https://www.npgsql.org/doc/release-notes/6.0.html#changes-to-ssl-configuration-ssl-moderequire](https://www.npgsql.org/doc/release-notes/6.0.html#changes-to-ssl-configuration-ssl-moderequire) ). This means that we cannot set the `SSL Mode` to `Require` any more, but instead we should use `SSL Mode=VerifyFull`.

The breaking change is related to **v6** of `Npgsql` so previous versions seem to work fine with `SSL Mode=Require`. Therefor it would be great with an "input override" option to specify what `SSL Mode` to use like the ones in [https://github.com/dfds/infrastructure-modules/tree/master/database/postgres#input-requirements-values-are-only-for-examples](https://github.com/dfds/infrastructure-modules/tree/master/database/postgres#input-requirements-values-are-only-for-examples).

The connection string that is assembled and stored in parameter store seems to be handled here: [https://github.com/dfds/infrastructure-modules/blob/66212b1bf45220c2ed34a8a381adb8091f59ccae/_sub/database/postgres/outputs.tf](https://github.com/dfds/infrastructure-modules/blob/66212b1bf45220c2ed34a8a381adb8091f59ccae/_sub/database/postgres/outputs.tf).

This would be much appreciated.

Thanks.

**Comment by avnes** [comment]
@jandr-dfds 

This has now been implemented in release 0.5.21 of infrastrcuture-modules:
https://github.com/dfds/infrastructure-modules/releases/tag/0.5.21

The default value will still be Require, but from your calling side you can override this by sending 

`ssl_mode = "VerifyFull"` into the postgres module.

**Comment by jandr-dfds** [comment]
Thank you @avnes, much appreciated.
